ksm/py_count.js

```diff
@@ -0,0 +1,12 @@
+function solution(s){
+    let count1 = 0;
+    let count2 = 0;
+    
+    for(let i=0; i<s.length; i++) {
+        if(s[i] === 'y' || s[i]==='Y') {
+            count1++;
+        }else if(s[i] === 'p' || s[i]==='P') {
+            count2++;            
+        }  
+    }return count1===count2;
```

Comment:
```suggestion
    for (let i = 0; i < s.length; i++) {
        if (s[i] === 'y' || s[i]==='Y') {
            count1++;
        } else if(s[i] === 'p' || s[i]==='P') {
            count2++;            
        }  
    }
    return count1 === count2;
```
- for과 ( 괄호 사이에는 띄어쓰기 마찬가지로 if/else문도 (, { 괄호 사이는 띄어쓰기가 기본 컨벤션입니다 :) 
- 코드의 논리적 흐름을 명확히 하기 위해 return문은 줄바꿈을 해주세요~
  - 이러한 사소한 습관이 코드의 가독성을 키워줍니다.